okhttp/src/test/java/okhttp3/OkPutHeadTest.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
```

Comment:
@swankjesse any thoughts on where to put specific tests like this? CallTest? or a directory for bug report tests?

Reply:
I don’t mind CallTest as a catch all for everything. There’s no satisfying way to organize unit tests like ours where we’re testing how pairs or triples of features interact with each other.

Reply:
We have CallKotlinTest, so I'll put kotlin tests there.